src/Utils/OrientationUtils.js

```diff
@@ -252,11 +361,21 @@ export default {
      * @param {string|Projection} crsOrProj - the CRS of the source frame or its
      * proj4-compatible object.
      *
-     * @return {OrientationFunction} the orientation function
+     * @param {Coordinates} [coordinates]  coordinates the origin of the local East North Up
+     * (ENU) frame
+     * @param {THREE.Quaternion} [target=new THREE.Quaternion()] output Quaternion
+     * @return {FunctionOrQuaternion} The target quaternion if coordinates is defined, otherwise, a function to compute it from coordinates.
      */
-    quaternionFromCRS(crsOrProj) {
-        const toCrs = this.quaternionToCRS(crsOrProj);
-        return (coordinates, target = new THREE.Quaternion()) => toCrs(coordinates, target).conjugate();
+    quaternionFromCRS(crsOrProj, coordinates, target = new THREE.Quaternion()) {
```

Comment:
May be replace by  `quaternionFromCoordinates` and so, and use Coordinates.crs to found `crsOrProj`.